docs/csharp/language-reference/configure-language-version.md

```diff
@@ -1,64 +1,69 @@
 ---
-title: Select the C# language version - C# Guide
-description: Configure the compiler to perform syntax validation using a specific compiler version
-ms.date: 02/28/2019
+title: C# language versioning - C# Guide
+description: Learn about how the C# language version is determined based on your project, and the different values you can manually adjust it to.
+ms.date: 07/10/2019
 ---
 
-# Select the C# language version
+# C# language versioning
 
-The C# compiler determines a default language version based on your project's target framework or frameworks. When your project targets a preview framework that has a corresponding preview language version, the language version used is the preview language version. When your project doesn't target a preview framework, the language version used is the latest minor version.
+The C# compiler determines a default language version based on your project's target framework or frameworks. This is because the C# language may have features that rely on types or runtime components that are not available in every .NET implementation. This also ensures that for whatever target your project is built against, you get the highest compatible language version by default.
 
-For example, during the preview period for .NET Core 3.0, any project that targets `netcoreapp3.0` or `netstandard2.1` (both in preview) will use the C# 8.0 language (also in preview). Projects targeting any released version will use C# 7.3 (the latest released version). This behavior means that any project targeting .NET Framework will use latest (C# 7.3). 
+## Defaults
 
-This capability decouples the decision to install new versions of the SDK and tools in your development environment from the decision to incorporate new language features in a project. You can install the latest SDK and tools on your build machine. Each project can be configured to use a specific version of the language for its build. The default behavior means that any language features that rely on new types or new CLR behavior are enabled only when projects target those frameworks.
+The compiler determines a default based on these rules:
 
-You can override the default behavior by specifying a language version. There are several ways to set the language version:
+|Target framework|version|C# language version default|
+|----------------|-------|---------------------------|
+|.NET Core|3.x|C# 8.0|
+|.NET Core|2.x|C# 7.3|
+|.NET Standard|all|C# 7.3|
```

Comment:
Why wouldn’t .NET Standard 2.1 select C# 8.0?

Reply:
Rather complicated reasons, but the long and short of it is there's no way to actually enforce any kind of default for any arbitrary target that implements a .NET Standard. Hence it is 7.3.